src/main/java/seedu/address/model/question/Description.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.question;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Question's description in the address book.
```

Comment:
I am not sure what to change the "address book" to, maybe "SETA" or "list of questions"